src/model/schema/ThunderstoreSchema.ts

```diff
@@ -0,0 +1,49 @@
+import Ajv from "ajv";
+import addFormats from "ajv-formats";
+
+import { ThunderstoreEcosystem }  from "../../assets/data/ecosystem.d";
+import ecosystem from "../../assets/data/ecosystem.json";
+import jsonSchema from "../../assets/data/ecosystemJsonSchema.json";
+import R2Error from "../errors/R2Error";
+
+
+export class EcosystemSchema {
+    private static _isValidated: boolean = false;
+
+    /**
+     * Get a validated instance of the ecosystem schema.
+     */
+    private static get get(): ThunderstoreEcosystem {
```

Comment:
Not a fan of this name, what is it getting?
If we rename this to: `thunderstoreEcosystem` so it becomes `this.thunderstoreEcosystem`

Reply:
The naming is leftover from when the method was public, so it read `ThunderstoreSchema.get`. I'm fine renaming it but I'll probably do it in a separate PR.